gulpfile.js

```diff
@@ -23,15 +23,15 @@ gulp.task('browser', function() {
     .pipe(gulp.dest('./Example/source'));
 });
 
-gulp.task('es6', function(){
+gulp.task('es6',_ => {
```

Comment:
@ManrajGrover Just a question . Why do you need browser task at all.[I think you did it for testing]

What we can do is make a test script and integrate travis with our repo this way whenever someone sends a PR or updates something.
Travis will run in background and check if all test cases will be passed or not by changed code.

In that test scripts we can check for our test cases and all.

Travis has a beautiful feature of encryption and hiding the keys and all .

Check [here](https://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-Repository-Settings)

I hope that would more or less solve your problem.